### PR TITLE
[docs] Add status header for SignalFx receiver

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** 
Add status header for SignalFx receiver

/cc @pjanotti @dmitryax as code owners to verify new status header

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

File: receiver/signalfxreceiver/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # SignalFx Receiver
 
+| Status                   |               |
+| ------------------------ |---------------|
+| Stability                | [beta]        |
```
Comment (pjanotti) [inline receiver/signalfxreceiver/README.md:5]: Could you please change it to `stable` instead?

**Review by pjanotti — CHANGES_REQUESTED** [review]
@Frapschen could you please mark this one as stable instead.

File: receiver/signalfxreceiver/README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+| Supported pipeline types | traces, logs  |
```
Comment (pjanotti) [inline receiver/signalfxreceiver/README.md:6]:
```suggestion
| Supported pipeline types | metrics, logs  |
```

File: receiver/signalfxreceiver/README.md (hunk 1 above)
Comment (Frapschen) [reply receiver/signalfxreceiver/README.md]: @pjanotti done

File: receiver/signalfxreceiver/README.md (hunk 2 above)
Comment (Frapschen) [reply receiver/signalfxreceiver/README.md]: doen

**Review by pjanotti — APPROVED** [review]
Thanks @Frapschen

**Review by codeboten — APPROVED** [review]

**Pull request merged**